flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java

```diff
@@ -326,4 +327,48 @@ public void testTwoSinksInDisjointedDAG() throws Exception {
     Assert.assertEquals("rightTable", rightTable.currentSnapshot().summary().get("direction"));
   }
 
+  @Test
+  public void testOverrideWriteConfiguration() {
```

Comment:
nit: make the test method name more descriptive. like `testOverrideWriteConfigWithUnknownDistributionMode`